docs/execution-providers/Xnnpack-ExecutionProvider.md

```diff
@@ -0,0 +1,83 @@
+---
+title: XNNPACK
+description: Instructions to execute ONNX Runtime with the XNNPACK execution provider
+parent: Execution Providers
+nav_order: 16
+redirect_from: /docs/reference/execution-providers/Xnnpack-ExecutionProvider
+---
+{::options toc_levels="2" /}
+
+# XNNPACK Execution Provider
+
+Accelerate ONNX models on Android devices and WebAssembly with ONNX Runtime and the XNNPACK execution provider. [(XNNPACK)](https://github.com/google/XNNPACK) is a highly optimized library of floating-point neural network inference operators for ARM, WebAssembly, and x86 platforms.
+
+## Contents
+{: .no_toc }
+
+* TOC placeholder
+{:toc}
+
+## Requirements
+
+
+## Install
+No Pre-built packages of ONNX Runtime with XNNPACK EP for Android are available.
+
```

Comment:
We include it in the full package don't we? 

[https://github.com/microsoft/onnxruntime/blob/b820256f34d94a4f58512dd6fcffdc7ab9a4df8a/tools/ci_build/github/android/default_full_aar_build_settings.json#L17](https://github.com/microsoft/onnxruntime/blob/b820256f34d94a4f58512dd6fcffdc7ab9a4df8a/tools/ci_build/github/android/default_full_aar_build_settings.json#L17)

Reply:
Yeah, we support xnnpack in full-build package, thanks